ontology/yaml/resources/entity_types/global.yaml

```diff
@@ -12,6 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+EQUIPMENT:
+  id: "9693662434551660544"
+  description: "A piece of equipment."
+  is_abstract: true
+  opt_uses:
+  - manufacturer_label
+  - model_label
+
+MTS:
+  description: "Multi-sensor device."
+  is_abstract: true
+  implements:
+  - EQUIPMENT
+
+USER_INTERFACE:
+  description: "Generic user interface."
+  is_abstract: true
+  implements:
+  - EQUIPMENT
+
 OS:
   id: "5263290281591046144"
```

Comment:
is this an occ sensor? a status is a subfield. would it also make sense to put this in a separate namespace? not sure what it would be, but is it really generic globally?